.travis.yml

```diff
@@ -14,4 +14,4 @@ script:
   # Tests
   - python setup.py test
   # pep8 - disabled for now until we can scrub the files to make sure we pass before turning it on
-  - pycodestyle .
+  - pycodestyle tableauserverclient test
```

Comment:
A recent update to pycodestyle (since we always install the latest in the travis run) is picking up files that are outside of our control (libraries installed), so now we want to limit to specifically code we have control over.